docs/acceptance_setup.md

```diff
@@ -0,0 +1,104 @@
+
+## Provision a Base VM
+
+These instructions cover provisioning a Ubuntu 16.04 VM:
+
+1. Download [a Ubuntu 16.04 64-bit ISO](http://releases.ubuntu.com/16.04/).
+2. Create a new VMWare Virtual Machine for it, named `beaker-ubuntu-1604-64`.
+3. Run the OS installation. Default settings should be adequate. Name the main user whatever you want; testing relies on passwordless root login so this is unimportant.
+4. Install and configure OpenSSH on the VM:
+    ~~~console
+    $ sudo su -
+    root@host $ apt-get install openssh-server
+    root@host $ nano /etc/ssh/sshd_config
+    # Add/uncomment/modify the following lines:
+    PasswordAuthentication yes
+    PermitRootLogin without-password
+    RSAAuthentication yes
+    PubkeyAuthentication yes
+    root@host $ service ssh restart
+    ~~~
+5. Disable automatic package updates so they don't block test running. This can be accomplished easily from the UI: System Settings > Software & Updates > Updates > Automatically check for updates: "Never".
+
+## Configure Testing VMs
+
+These instructions cover provisioning the two testing VMs with a new public key for SSH access. We need two instances of the base VM, `beaker-ubuntu-1604-64` and `beaker-ubuntu-1604-64-2`:
+
+1. You can use the base VM configuration as the first SUT to save space.
+2. Configure a static IP for the VM. See [fusion_static_ip.md](fusion_static_ip.md).
+3. Add an entry in /etc/hosts: `<static ip> beaker-ubuntu.local`.
+4. Set Up password-less SSH access:
+    - Generate a new SSH key *without a passphrase*:
+        ~~~console
+        $ ssh-keygen -t rsa -b 4096 -C "you@example.com"
+        # Give this a descriptive name, e.g. "id_rsa-beaker-ubuntu"
+        # Make sure to leave passphrase blank (just hit return at prompt)
+        ~~~
+    - Add it to your local keychain:
+        ~~~console
+        $ ssh-add -i ~/.ssh/id_rsa-beaker-ubuntu
+        ~~~
+    - Add your SSH key to the VM's authorized keys:
+        ~~~console
+        $ ssh-copy-id puppetlabs@beaker-ubuntu.local
+        $ ssh puppetlabs@beaker-ubuntu.local
+        ~~~
+    - And then copy to root's `authorized_keys`:
+        ~~~console
+        $ ssh puppetlabs@beaker-ubuntu.local
+        puppetlabs@beaker-ubuntu.local $ sudo cp ~/.ssh/authorized_keys /root/.ssh/authorized_keys
+        ~~~
+    - Ensure that you can SSH in without password, e.g.:
+        ~~~console
+        $ ssh root@beaker-ubuntu.local
+        ~~~
+5. Create a new snapshot (of the running VM) in VMWare Fusion: with the VM selected, Virtual Machine > Snapshots... > Take Snapshot (Cmd-Shift-S). Name this new snapshot `clear-keys`. This snapshot name is used in [acceptance/config/nodes/test-nodes.yml](acceptance/config/nodes/test-nodes.yml); when beaker provisions a VMWare host it restores it to a specified snapshot.
```

Comment:
still broken. I suspect that adding a `/` to the beginning of the link might fix the issue. Looks like the relative link will include the `docs` folder since it's where this doc lives, which breaks the hosts file URL.